addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/core/LxCategory.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.loxone.core;
```

Comment:
Do you thing it can reside as an exported package inside org.openhab.binding... so other projects can start depending on it?
I would like to avoid creating a separate project and package and import it into openhab, as this is really the only use case for me, but I think the community may benefit from such package. It would be great to allow people to use it as a package, not to fork/copy the source code, as I can see already happening now.
But would that be something ok from architectural point of view?